non-free.md

```diff
@@ -119,6 +119,10 @@
 - [RhodeCode](https://rhodecode.com) `⊘ Proprietary` - On-premise Source Code Management for Mercurial, Git & Subversion. `Python`
 - [BitBucket Server](https://www.atlassian.com/software/bitbucket/server) `⊘ Proprietary` - An enterprise-level Git solution similar to GitLab `Java`
   
+### Software Development - Security & QA
+
+- [Betterscan CE (Community Edtion)](https://www.betterscan.io/) - Code Scanning/SAST/Static Analysis/Linting using many tools/Scanners with One Report - Betterscan Community Edition (CE). ([Source Code](https://github.com/marcinguy/betterscan-ce)) `BSD-3-Clause LGPL-2.1 Commonsclause` `Python`
```

Comment:
Any category you are creating has the minimum requirement of 3 items.
I suggest adding it `Software Development` instead.